pydicom/_version.py

```diff
@@ -5,3 +5,8 @@
 __version__ = '2.1.0.dev0'
 __version_info__ = tuple(
     re.match(r'(\d+\.\d+\.\d+).*', __version__).group(1).split('.'))
+
+
+# DICOM Standard version used for:
+#   _dicom_dict, _uid_dict and _storage_sopclass_uids
+__dicom_version__ = '2020d'
```

Comment:
I don't see _pydicom's_ `__init__.py` updated in this PR to bring it in, so can do `pydicom.__dicom_version__`, similar to `pydicom.__version__`